DEVELOPER-GUIDE.md

```diff
@@ -0,0 +1,47 @@
+Developer Guide
+===
+
+- [1. Coding Guidelines](#1-coding-guidelines)
+    - [1.1. General](#11-general)
+    - [1.2. Usage of `var`](#12-usage-of-var)
+    - [1.3. Working with `async` methods](#13-working-with-async-methods)
+    - [1.4. Testing](#14-testing)
+
+# 1. Coding Guidelines
+
+## 1.1. General
+- Every public field/method should be documented with XML docs
+- One file should only contain one class
```

Comment:
Also not always a fan of this, makes it sometimes harder to grasp how an application is structured, but for Arcus, alright.

Reply:
That's how it is anyhow so guess it's ok, no? :D

Reply:
Resolving as this is our Arcus guideline